cmd/cycloid/middleware/kpi.go

```diff
@@ -0,0 +1,110 @@
+package middleware
+
+import (
+	"github.com/cycloidio/cycloid-cli/client/client/organization_kpis"
+	"github.com/cycloidio/cycloid-cli/client/models"
+	"github.com/cycloidio/cycloid-cli/cmd/cycloid/common"
+	strfmt "github.com/go-openapi/strfmt"
+	"encoding/json"
+
+)
+
+
+func (m *middleware) CreateKpi(name, kpiType, widget, org, project, job, env, config string) (*models.KPI, error) {
+	var body *models.NewKPI
+	var err error
+
+	params := organization_kpis.NewCreateKpiParams()
+	params.SetOrganizationCanonical(org)
+	pipeline := ""
+	if project != "" {
+			params.SetProject(&project)
```

Comment:
looks a bit over indented ? did you `go fmt` or `goimports` ?

Reply:
I will do it.
I saw this kind of issue in other files so I also plan to run a go imports on all files in the error handleing PR as I change nearly all files.